.github/workflows/publish.yml

```diff
@@ -29,6 +29,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_PAT }}
 
+      - name: Bump Homebrew Core
+      - uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
```

Comment:
I think we also need to add the `version` right?

Reply:
I think it'll just pick the ref?

<img width="407" alt="CleanShot 2022-08-03 at 13 51 01@2x" src="https://user-images.githubusercontent.com/5499880/182709214-d01bac9f-7466-4831-84fd-a116211227d7.png">

Which is what we do here right?

<img width="452" alt="CleanShot 2022-08-03 at 13 51 20@2x" src="https://user-images.githubusercontent.com/5499880/182709294-71eccbf5-4136-476d-a7fb-f560ec0eb148.png">

Reply:
Oh perhaps. Not really sure tbh. I was just thinking we do use the version manually here https://github.com/railwayapp/cli/blob/master/.github/workflows/publish.yml#L65.

But can give it a go like this.